trade-finance-manager-api/src/v1/mappings/map-ukef-documents-to-estore.js

```diff
@@ -36,16 +37,21 @@ const eStoreMap = {
   },
 };
 
-const mapUKEFDocumentsToEstore = (files) => {
+/**
+ * Produces an array of files mapped per eStore consumption
+ * @param {Object} files Object of files uploaded
+ * @returns Mapped array of files for eStore API consumption
+ */
+const mapEstoreFiles = (files) => {
   const documents = [];
 
   const nonDocuments = ['validationErrors', 'security', 'status', 'securityDetails', 'requiredFields'];
   Object.keys(files).forEach((file) => {
     if (!nonDocuments.includes(file)) {
       Object.values(files[file]).forEach((val) => {
-        if (eStoreMap[file]?.eStoreName) {
+        if (estoreFiles[file]?.eStoreName) {
```

Comment:
Could use lodash's `has` method here

```suggestion
        if (has(estoreFiles, file)) {
```